managed/services/agents/postgresql.go

```diff
@@ -34,6 +35,7 @@ var (
 	postgresExporterAutodiscoveryVersion = version.MustParse("2.15.99")
 	postgresExporterWebConfigVersion     = version.MustParse("2.30.99")
 	postgresSSLSniVersion                = version.MustParse("2.40.99")
+	postgresMaxExporterConnsVersion      = version.MustParse("2.41.99")
```

Comment:
it will be in 2.41.2 or 2.41.3, so I think better to use version `2.41.2-0`

Reply:
Done

TODO: double check before merge